parser/src/ast.rs

```diff
@@ -391,3 +391,28 @@ pub enum StringGroup {
         values: Vec<StringGroup>,
     },
 }
+
+#[derive(Debug, PartialEq)]
+pub enum Varargs {
+    None,
+    NoCapture,
+    Capture(Parameter),
+}
+
+impl Default for Varargs {
+    fn default() -> Varargs {
+        Varargs::None
+    }
+}
+
+impl From<Option<Option<Parameter>>> for Varargs {
```

Comment:
Where's the double `Option` coming from? Is there a way to eliminate it altogether?

Reply:
I didn't really want to mess around in the `python.lalrpop` grammar, as I don't really have any experience with it, so I opted for this instead.

Reply:
Okay, that's fine, was just wondering

Reply:
The double option is a sign of a triple option enum. Actually the options are: namedparameter, namedstar, and unnamed star.